src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/product.yml

```
@@ -25,3 +25,33 @@ admin_products_v2_edit:
       id_product: productId
   requirements:
     productId: \d+
+
+admin_products_v2_get_images:
+    path: /{productId}/getImages
```

```suggestion
    path: /{productId}/get-images
```

We should be consistent with other routes, like `bulk-enable-status`, `send-testing-email`, `bulk-delete`, etc.